app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -4,19 +4,21 @@ child :content_facet => :content_facet_attributes do
   node do |content_facet|
     content_facet.single_content_view # just so Rails gives a warning and we remember to fix this
     version = content_facet.content_view_environments.first.content_view_version
```

Comment:
I'm guessing `content_facet.content_view_environments.first` can be `nil` too. Not sure if `content_facet.content_view_environments` is always a list of even that can be `nil`.
```suggestion
    version = content_facet.content_view_environments.first&.content_view_version
```